atlas/__mocks__/svgrMock.js

```diff
@@ -0,0 +1,4 @@
+import * as React from 'react'
+export default 'SvgrURL'
+const SvgrMock = React.forwardRef((props, ref) => <span ref={ref} {...props} />)
```

Comment:
We have a linting issue here, we should start enforcing ESLint now so we need to either fix the issue or revise the rules behind it